compiler/src/parsing/lexer.re

```diff
@@ -113,7 +116,9 @@ let dec_float = [%sedlex.regexp?
 
 let unsigned_float = [%sedlex.regexp? dec_float];
 
-let uident = [%sedlex.regexp? (lu, Star(xid_continue))];
+let uident = [%sedlex.regexp?
+  (Intersect(xid_start, lu), Star(xid_continue))
```

Comment:
This is equivalent to what it was before, but I felt it made sense to make it more explicit since it's not obvious that `Lu` is a subset of `xid_start`.